src/ConfigCatClientOptions.ts

```diff
@@ -2,21 +2,31 @@ import { ConfigCatConsoleLogger } from "./ConfigCatLogger";
 import { IConfigCatLogger, IAutoPollOptions, ILazyLoadingOptions, IManualPollOptions, LogLevel } from "./index";
 import COMMON_VERSION from "./Version";
 
+
+/** Restrict the location of your feature flag and setting data within the ConfigCat CDN. */
 export enum DataGovernance {
+    /** Your data will be published to all ConfigCat CDN nodes to guarantee lowest response times. */
     Global = 0,
+    /** Your data will be published to CDN nodes only in the EU. */
```

Comment:
If your data is published to CDN nodes only in the EU.

Reply:
done